### PR TITLE
Implement get_tags for GithubProject

### LINKED ISSUE #303: get_tags is not implemented for GithubProject
```
ipdb> git_project                                                                                                                                                            
<ogr.services.github.project.GithubProject object at 0x7f7314f47510>

ipdb> git_project.get_tags                 
<bound method GitProject.get_tags of <ogr.services.github.project.GithubProject object at 0x7f7314f47510>>

ipdb> git_project.get_tags()               
*** NotImplementedError
```

### PR DESCRIPTION
**Pull request by mfocko**
Fixes #303

### CODE REVIEW

File: tests/integration/test_github.py [hunk 1]
```diff
@@ -259,6 +259,22 @@ def test_get_tag_from_tag_name(self):
     def test_get_tag_from_nonexisting_tag_name(self):
         assert not self.ogr_project.get_tag_from_tag_name("future")
 
+    def test_get_tags(self):
+        tags = self.ogr_project.get_tags()
+
+        names = {f"0.{i}.0" for i in range(1, 10)}
+        names.update({"0.0.1", "0.0.2", "0.0.3", "0.3.1"})
+        assert names == set(map(lambda tag: tag.name, tags))
```
Comment (lachmanfrantisek) [inline tests/integration/test_github.py]: Isn't this a bit fragile? We will break this test on each ogr release.

Comment (mfocko) [reply tests/integration/test_github.py]: should be fixed now

Comment (lachmanfrantisek) [reply tests/integration/test_github.py]: Thanks!

**Pull request merged**